examples/platform/silabs/LEDWidget.cpp

```diff
@@ -82,3 +81,21 @@ void LEDWidget::Animate()
         }
     }
 }
+
+uint8_t LEDWidget::GetLED()
+{
+    return mLed;
+}
+
+bool LEDWidget::GetLEDStatus(uint8_t led)
+{
+    return mLedStatus;
+}
```

Comment:
still no space between function here and at uint8_t LEDWidget::GetLevel()

Reply:
Added.